scripts/complete_liquidity_provision.js

```diff
@@ -128,6 +129,19 @@ module.exports = async (callback) => {
           }
         })
       )
+      // Detect if provided brackets have existing orders.
+      const existingOrders = await Promise.all(
+        bracketAddresses.map(async (safeAddr) => {
+          return await hasExistingOrders(safeAddr, exchange)
```

Comment:
Nitpick: no await needed here.
```suggestion
          return hasExistingOrders(safeAddr, exchange)
```